### PR TITLE
Update Makefile & Add Firebase In-App Message

### PR DESCRIPTION
**Pull request by jgsamudio**
# 📲 What

Updates the makefile to fix the `carthage-bootstrap` command. Also adds the Firebase dependency for the In-App messaging.

# 🤔 Why

We need the ability to use the Makefile to setup carthage dependencies

### CODE REVIEW

File: Cartfile.resolved [hunk 1]
```diff
@@ -1,8 +1,9 @@
 binary "https://dl.google.com/dl/firebase/ios/carthage/FirebaseAnalyticsBinary.json" "7.4.0"
 binary "https://dl.google.com/dl/firebase/ios/carthage/FirebaseCrashlyticsBinary.json" "7.4.0"
+binary "https://dl.google.com/dl/firebase/ios/carthage/FirebaseInAppMessagingBinary.json" "7.4.0"
 binary "https://dl.google.com/dl/firebase/ios/carthage/FirebasePerformanceBinary.json" "7.4.0"
 binary "https://dl.google.com/dl/firebase/ios/carthage/FirebaseProtobufBinary.json" "7.4.0"
-binary "https://raw.githubusercontent.com/PerimeterX/px-iOS-Framework/master/PerimeterX.json" "1.13.8"
+binary "https://raw.githubusercontent.com/PerimeterX/px-iOS-Framework/master/PerimeterX.json" "1.13.9"
```
Comment (singhhari) [inline Cartfile.resolved:6]: I'm assuming this updated after you called `carthage update` ?

**Review by singhhari — APPROVED** [review]

File: Cartfile.resolved (hunk 1 above)
Comment (jgsamudio) [reply Cartfile.resolved:6]: Yes! I just updated the cartfile to pin the perimeter x version

File: Makefile [hunk 2]
```diff
@@ -42,7 +42,7 @@ dependencies: carthage-bootstrap configs secrets
 bootstrap: hooks dependencies
 
 carthage-bootstrap:
-	bin/carthage.sh || exit 1
+	bin/carthage.sh bootstrap --platform iOS || exit 1
```
Comment (justinswart) [inline Makefile:45]: @jgsamudio catching up here - strange, this should have been handled within the script itself based on whether it's running on CI or not 🤔

Comment (jgsamudio) [reply Makefile:45]: Yeah I was running into issues with this script where I needed to specify bootstrap and the platform. I agree that its odd that it didn't run the command for local builds

**Pull request merged**